pasqal_cloud/__init__.py

```diff
@@ -131,35 +128,30 @@ def create_batch(
         if configuration:
             req.update({"configuration": configuration.to_dict()})  # type: ignore
 
-        batch_rsp, jobs_rsp = self._client._send_batch(req)
+        batch_rsp = self._client._send_batch(req)
         batch_id = batch_rsp["id"]
-        if wait or fetch_results:
+        if wait:
             while batch_rsp["status"] in ["PENDING", "RUNNING"]:
                 time.sleep(RESULT_POLLING_INTERVAL)
-                batch_rsp, jobs_rsp = self._client._get_batch(batch_id)
+                batch_rsp = self._client._get_batch(batch_id)
 
-            if fetch_results:
-                batch_rsp, jobs_rsp = self._client._get_batch(
-                    batch_id, fetch_results=True
-                )
-        batch = Batch(**batch_rsp, jobs=jobs_rsp, _client=self._client)
+        batch = Batch(**batch_rsp, _client=self._client)
```

Comment:
Are the jobs also returned by the get batch endpoint ?

Reply:
yes